app/forms/support_interface/country_form.rb

```diff
@@ -81,4 +87,31 @@ def diff_actions
         (delete_actions + create_actions).sort_by { |action| action[:name] }
       end
   end
+
+  def eligibility_route
+    if subject_limited
+      "expanded"
+    elsif eligibility_skip_questions
+      "reduced"
+    else
+      "standard"
+    end
+  end
+
+  def eligibility_route=(value)
+    subject_limited_will_change!
+    eligibility_skip_questions_will_change!
```

Comment:
Do we need these methods on this form? I know we needed it when the code was on the model directly, but I wonder if we need it on the form?

Reply:
Doesn't seem to be needed. Everything works without these methods